lib/Event.ts

```diff
@@ -146,29 +147,35 @@ function dataToCard(data: EventData): EventCard {
  *                is used as a cache key.
  */
 export async function getEventCards(count: number): Promise<EventCard[]> {
+    // // !!OLD CODE USING NEXT JS CACHE SYSTEM!!
```

Comment:
Any reason to keep this in the code base? Can we just remove it?

Reply:
kept just in but I'll remove it :D